src/cloud/shared/messages/nats_metrics.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package messages
+
+import (
+	"strings"
+
+	"github.com/nats-io/nats.go"
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+// NatsErrorCount counts the number of errors that occurs over NATs.
```

Comment:
I think it would be a bit cleaner to wrap this in a struct, something like:
```
type NatsErrorCounter struct {
   count *prometheus.CounterVec
}

func NewNatsErrorCounter() {
  c := ....
  prometheus.MustRegister(c.count)
}

func (c *NatsErrorCounter) HandleNatsError
```
And then each server can create a new nats error counter in its init function, this way the global state isn't in the shared file.